tests/test_gpu_examples.py

```diff
@@ -518,3 +524,199 @@ def make_inputs_require_grad(module, input, output):
 
             # assert loss is not None
             self.assertIsNotNone(trainer.state.log_history[-1]["train_loss"])
+
+
+@require_torch_gpu
+@require_auto_gptq
+@require_optimum
+class PeftGPTQGPUTests(unittest.TestCase):
```

Comment:
Can we also get a seq2seq test or do they not work yet?

Reply:
seq2seq test are not supported yet as this quantization method only works on decoder only or encoder only architecture.